Added Calendar WeekView, MonthView

alexanderDracka opened the pull request:
MonthView, WeekView, DaysView, TimezoneSupport, Drag/Resize Validation examples

alexanderDracka commented:
Fixed WeekView example, merged upstream changes.

alexanderDracka commented:
Added Days View and Timezone Support examples. Please @markbrocato look at the Timezone support closely as I am not sure if I chose correct approach in changeOptions method. It is approved way how to change config?

markbrocato commented on packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js:
```diff
@@ -0,0 +1,73 @@
+    constructor(){
+        super();
+    }
+    changeOptions(){
```
use changeOptions = () => { ... } instead so you don't need to do this.changeOptions.bind(this) later.

markbrocato commented on packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js:
```diff
@@ -0,0 +1,73 @@
+    }
+    changeOptions(){
+        const selectorValue = Ext.getCmp('selector').getSelection().data.value;
+        this.refs.timezoneview.setConfig('timezoneOffset', selectorValue)
```
This should be replaced with this.setState({ timezoneOffset: selectorValue }) instead.

markbrocato commented on packages/reactor-kitchensink/src/examples/Calendar/TimezoneSupport.js:
```diff
@@ -0,0 +1,73 @@
+        }
+    }
+
+    calendarTimezoneConfig={
```
This should go in the render method as the props for <Calendar_Day>, timezoneOffset should be set to {this.state.timezoneOffset}.

markbrocato commented on packages/reactor-kitchensink/src/examples/Calendar/WeekView.js:
```diff
@@ -0,0 +1,83 @@
+        super();
+    }
+
+    changeToWorkWeek(){
```
Use ES2017 property initializer changeToWorkWeek = () => {...}

alexanderDracka commented:
@markbrocato sorry for these commits as I should made separate PR. Will do next time.
Added Fix for EdgeMenu and removed one deprecated config in Basic markers
